core/src/main/java/edu/wpi/grip/core/events/ExceptionEvent.java

```diff
@@ -22,7 +23,7 @@
      * @param message   The message associated with this event.
      *                  If <tt>null</tt> will use {@link Exception#getMessage()}
      */
-    public ExceptionEvent(Object origin, Exception exception, String message) {
+    public ExceptionEvent(Object origin, @Nullable Exception exception, @Nullable String message) {
```

Comment:
Ok.  I would wait and do that change in a different PR so everyone has a chance to weigh in on whether or not to use it.